<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskWidget</class>
 <widget class="QWidget" name="taskWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="taskGrpBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox { background: rgba(250, 255, 250, 1); border: 1px solid; border-radius: 5px; border-color: rgb(146, 146, 146); margin: 8px;}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 2px;
    margin: 0;
}</string>
     </property>
     <property name="title">
      <string>Task</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" name="taskLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="wndLabel">
          <property name="text">
           <string>Set Window to</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="3">
         <widget class="QLineEdit" name="cwdPath">
          <property name="placeholderText">
           <string>Current Working Directory</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QCheckBox" name="allowMultiple">
          <property name="text">
           <string>Allow mutiple instances</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="processLabel">
          <property name="text">
           <string>Executable Path</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="commandLabel">
          <property name="text">
           <string>Command</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="processPathBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QPushButton" name="testBtn">
          <property name="text">
           <string>Test Run</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="processPath">
          <property name="placeholderText">
           <string>Executable to launch</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="cwdLabel">
          <property name="text">
           <string>CWD</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="5">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item alignment="Qt::AlignLeft">
           <widget class="QCheckBox" name="wndMinCheckBox">
            <property name="text">
             <string>minimized</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QCheckBox" name="wndActCheckBox">
            <property name="text">
             <string>active</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="2" colspan="3">
         <widget class="QLineEdit" name="commandLine">
          <property name="placeholderText">
           <string>additional Arguments</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QComboBox" name="commandStop">
          <item>
           <property name="text">
            <string>start</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>stop</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="taskDelete">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/main/delete-24px.svg</normaloff>:/main/delete-24px.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="4" column="6">
         <widget class="QToolButton" name="cwdPathBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="taskLabelLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="taskIconLabel">
            <property name="maximumSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="res.qrc">:/main/widgets-24px.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="taskLabel">
            <property name="text">
             <string>Task</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2" colspan="3">
         <widget class="QLineEdit" name="taskName">
          <property name="placeholderText">
           <string>Task Name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="conditionBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>42</height>
         </size>
        </property>
        <property name="title">
         <string>Conditions</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="addConditionBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/main/add-24px.svg</normaloff>:/main/add-24px.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>taskName</tabstop>
  <tabstop>taskDelete</tabstop>
  <tabstop>processPath</tabstop>
  <tabstop>processPathBtn</tabstop>
  <tabstop>allowMultiple</tabstop>
  <tabstop>commandStop</tabstop>
  <tabstop>commandLine</tabstop>
  <tabstop>testBtn</tabstop>
  <tabstop>cwdPath</tabstop>
  <tabstop>cwdPathBtn</tabstop>
  <tabstop>wndMinCheckBox</tabstop>
  <tabstop>wndActCheckBox</tabstop>
  <tabstop>addConditionBtn</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
